src/client/src/apps/MainRoute/widgets/spotTile/SpotTileContainer.tsx

```diff
@@ -27,7 +27,8 @@ export interface SpotTileContainerOwnProps {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch, ownProps: SpotTileContainerOwnProps) => ({
-  onMount: () => dispatch(SpotTileActions.subscribeToSpotTile(ownProps.id)),
+  onCurrencyPairChanged: (currencyPair: string) =>
```

Comment:
removed `onMount` in favour of `onCurrencyPairChanged` which is more explicit in saying what does it do

Reply:
and also it covers both scenarios - component being mounted and changes in props